### DISCUSSION TITLE
Cloudflared LXC

### DISCUSSION THREAD

**Discussion by jcastro** [tteck/Proxmox#3282] [Help Needed]
Hi!
I just installed Cloudflared LXC but there's no further instructions on how to set it up or where are the config files, any ideas? thanks!

**Comment by HWiese1980** [comment]
This. There doesn't even seem to be a cloudflared installed in the container. It feels like an empty debian container, nothing more. How to set this up as intended?

**Reply by digitaljdr** [reply]
These should help 

Manage from cloudflare access portal
https://developers.cloudflare.com/cloudflare-one/connections/connect-networks/get-started/create-remote-tunnel/

manage locally on the LXC 
https://developers.cloudflare.com/cloudflare-one/connections/connect-networks/get-started/create-local-tunnel/
